TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -31,23 +30,72 @@ private FullSelect CreateFullSelect(CodeElementsParser.FullselectContext context
             SubSelect subSelect = null;
             if (context.subselect() != null)
             {
-                subSelect = CreateSubSelect(context.subselect());
+                 subSelect = CreateSubSelect(context.subselect());
             }
-
             return new FullSelect(subSelect);
         }
-
         private SubSelect CreateSubSelect(CodeElementsParser.SubselectContext context)
         {
             FromClause fromClause = null;
             SelectClause selectClause = null;
             if (context.sql_selectClause() != null)
             {
                 selectClause = CreateSelectClause(context.sql_selectClause());
+                fromClause = CreateFromClause(context.from_clause());
```

Comment:
This is misplaced, just like we check whether `sql_selectClause()` is null or not before building the `SelectClause`, we have to check whether `from_clause()` is null or not so move this assignment out of the existing `if` and wrap it into its own `if`.